### ISSUE TITLE
dblclick event not firing in desktop Chrome when in device simulation mode

### ISSUE THREAD

**Issue by CramericaIndustries** [Leaflet/Leaflet#6777]
When you have the viewport meta tag with a value of "user-scalable=no" present in your html file:
`<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">`
then the map.on("dblclick") event stops firing as soon as as you switch Chrome dev tools to mobile mode (CTRL+SHIFT+M).
![Untitled-1](https://user-images.githubusercontent.com/5860703/63647548-2fb9cf80-c723-11e9-903a-af4408d77906.png)

Minimal example: [Plunker](https://plnkr.co/edit/StAuNUkPuQznHman9H0g?p=preview)

As soon as you remove the `user-scaleable=no` string from the viewport meta tag the dblclick event is being fired again.

**Environment**
- Leaflet version: 1.5.1+build.2e3e0ffb
- Browser (with version): Chrome Version 76.0.3809.100 (Official Build) (64-Bit)
- OS/Platform (with version): Windows 10 1809

**Example code**
```
<!DOCTYPE html>
<html>
	<head>
		<title>Quick Start - Leaflet</title>

		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">

		<link rel="shortcut icon" type="image/x-icon" href="docs/images/favicon.ico" />
	    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.5.1/dist/leaflet.css" integrity="sha512-xwE/Az9zrjBIphAcBb3F6JVqxf46+CDLwfLMHloNu6KEQCAWi6HcDUbeOfBIptF7tcCzusKFjFw2yuvEpDL9wQ==" crossorigin=""/>

	    
	   <script src="https://unpkg.com/leaflet@1.5.1/dist/leaflet.js" integrity="sha512-GffPMF3RvMeYyc1LWMHtK8EbPv0iNZ8/oTtHPx9/cc2ILxQ+u905qIwdpULaqDkyBKgOaB57QTMg7ztg8Jm2Og==" crossorigin=""></script>

	</head>


	<body>
		<div id="mapid" style="width: 600px; height: 400px;"></div>
		<script>

			var mymap = L.map('mapid',{
				// doubleClickZoom: false
			}).setView([51.505, -0.09], 13);

			L.tileLayer('https://api.tiles.mapbox.com/v4/{id}/{z}/{x}/{y}.png?access_token=pk.eyJ1IjoibWFwYm94IiwiYSI6ImNpejY4NXVycTA2emYycXBndHRqcmZ3N3gifQ.rJcFIG214AriISLbB6B5aw', {
				maxZoom: 18,
				attribution: 'Map data &copy; <a href="https://www.openstreetmap.org/">OpenStreetMap</a> contributors, ' +
					'<a href="https://creativecommons.org/licenses/by-sa/2.0/">CC-BY-SA</a>, ' +
					'Imagery © <a href="https://www.mapbox.com/">Mapbox</a>',
				id: 'mapbox.streets'
			}).addTo(mymap);

			mymap.on("click",function(ev){
				console.log("click");
			}).on("dblclick",function(ev){
				console.log("dblclick");
				alert("dblclick");
			});
		</script>
	</body>


</html>

```

**Comment by sashadev-sky** [comment]
this is what im currently doing to get around this - just wrote this havent merged it yet for my plugin. (oe = originalEvent)
```JS
// manually fire doubleclick event only for touch screens
        if (L.Browser.touch && L.Browser.chrome) {
          if (!L.Browser.pointer) {
            map.fire('dblclick');
          } else if (oe && oe.sourceCapabilities.firesTouchEvents) {
            map.fire('dblclick');
          }
        }
```

and if you need the `e` variable to have all of the event properties they usually do in Leaflet:
```JS
 // manually fire doubleclick event only for touch screens
        if (L.Browser.touch) {
          if (!L.Browser.pointer) {
            // in `DoubleClickLabels.js`, we just do map.fire('dblclick') bc `_onDoublClick` doesn't use the
            // passed "e" (for now). To generate a 'real' DOM event that will have all of its corresponding core
            // properties (originalEvent, latlng, etc.), use Leaflet's `#map._fireDOMEvent` (Leaflet 1.5.1 source)
            map._fireDOMEvent(oe, 'dblclick', [map]);
          } else if (oe && oe.sourceCapabilities.firesTouchEvents) {
            map._fireDOMEvent(oe, 'dblclick', [map]);
          }
        }
```

Let me know what you think!

**Comment by johnd0e** [comment]
@CramericaIndustries
I recent Chrome I do not see any difference in behavior after removing "user-scalable=no" from your planker.

**Comment by mourner** [comment]
Likely to have been fixed by #7027?
